src/languages/tsx.coffee

```diff
@@ -2,7 +2,7 @@ module.exports = {
 
   name: "TSX"
   namespace: "tsx"
-  fallback: ['ts']
+  fallback: ['js']
```

Comment:
What is this for?

Reply:
The options defined here https://github.com/szeck87/atom-beautify/blob/add-prettier/src/beautifiers/typescript-formatter.coffee#L12-L15 aren't recognized if the fallback is to `ts`.